data/856/706/99/85670699.geojson

```diff
@@ -30,8 +30,10 @@
     "mz:max_zoom":11.0,
     "mz:min_zoom":7.7,
     "name:ara_x_preferred":[
-        "Bechar",
-        "\u0648\u0644\u0627\u064a\u0629 \u0628\u0634\u0627\u0631"
+        "\u0628\u0634\u0627\u0631"
+    ],
+    "name:ara_x_variant":[
```

Comment:
This is a latin script variant, do we have special property name for that?

Reply:
Resolved in 9c20bbd.. I edited this name, storing it in a `name:ara_latn_x_preferred` property.